common/src/main/java/com/wynntils/gui/render/CustomRenderType.java

```diff
@@ -8,14 +8,17 @@
 import com.mojang.blaze3d.vertex.VertexFormat;
 import com.mojang.blaze3d.vertex.VertexFormat.Mode;
 import java.util.OptionalDouble;
+import java.util.function.Function;
+import net.minecraft.Util;
 import net.minecraft.client.renderer.RenderStateShard;
 import net.minecraft.client.renderer.RenderType;
+import net.minecraft.resources.ResourceLocation;
 
 public class CustomRenderType extends RenderType {
     // Copied from RenderType.LINE_STRIP and changed the line width from the default
     // to 3
     public static final RenderType LOOTRUN_LINE = RenderType.create(
-            "lootrun",
+            "wynntils_lootrun_line",
             DefaultVertexFormat.POSITION_COLOR_NORMAL,
```

Comment:
What does the string literal do?

Reply:
Some sort of id, I just added wynntils if there is a problem with clashing